### ISSUE TITLE
[Bug]: "Encounter Type" in "Add a Condition" = TMI

### ISSUE THREAD

**Issue by phobrla** [fastenhealth/fasten-onprem#370]
### Contact Details

phil@hobrla.com

### What happened?

"Encounter" has way too many options. Can they be pared down to the values in DOC_EVENT_LIST_INFO.EVENT_ENC_TYPE_C_NAME at https://open.epic.com/EHITables/GetTable/DOC_EVENT_LIST_INFO.htm

- Registration
- Walk-In
- Hospital Encounter
- Canceled
- Unmerge
- Contact Moved
- Intake
- EMPTY
- Research Encounter
- Recurring Plan
- Update
- PCP/Clinic Change
- Wait List
- Enrollment
- Social Care Application
- Service Decision Registration
- Leader Rounds
- Clerical Orders
- Mother Baby Link
- Lactation Encounter
- Appointment
- Surgery
- Anesthesia
- Anesthesia Event
- Guardian Screening
- Ancillary Procedure
- Anticoagulation Discharge Documentation
- Health Maintenance Letter
- Patient Message
- E-Visit
- Mobile Order Only
- Questionnaire Series Submission
- Travel
- Patient Self-Triage
- Community Care Management
- Patient Outreach
- Telephone
- Nurse Triage
- E-Consult
- E-Consult Community Order
- Clinical Documentation Only
- Telemedicine
- Plan of Care Documentation
- External Communication
- Reconciled Outside Data
- External Contact
- Eye Exam
- Recurring Billing Documentation
- Pre-Admission Testing
- Episode Documentation Update
- Hospice Admission
- Home Infusion
- Home Infusion Billing
- Episode Update
- Home Health Resumption of Care Planning
- Home Health Admission
- Home Care Visit
- Home Care Update
- Patient Web Update
- Community Orders
- Committee Review
- Post Mortem Documentation
- Billing Encounter
- Lab Requisition
- Office Visit
- Consent Form
- Procedure Pass
- External Hospital Admission
- Letter (Out)
- Hospital
- Refill
- Immunization
- History
- Referral
- Orders Only
- Rx Refill Authorize
- Meds Only (Web)
- Meds Void (Web)
- Episode Changes
- Ancillary Orders
- Pharmacy Visit
- BPA
- Abstract

The current values in the "Encounter" should be placed in a "Reason for Visit" field.

### Flavor

Desktop

### Version

dev

**Comment by AnalogJ** [comment]
agreed, this is a much more precise list of encounter types than the one I'm using - https://www.hl7.org/fhir/R4/valueset-service-type.html 

However, I'm trying to use a list that is related to the [Encounter specification](https://www.hl7.org/fhir/R4/encounter.html) - or at the very least is commonly used & has associated medical codes

---

Might be worth it to just replace my unwieldy list with yours for now, and then try to find associated codes after the fact.
